### PR TITLE
feat(statics): add deprecated feature to gteth

### PR DESCRIPTION
**Pull request by ranger2255**
Ticket: COIN-622

### CODE REVIEW

File: modules/statics/src/base.ts [hunk 1]
```diff
@@ -292,6 +292,10 @@ export enum CoinFeature {
    * This coin is onboarded on etheruem rollup chain
    */
   ETH_ROLLUP_CHAIN = 'eth-rollup-chain',
+  /**
+   * This coin is hidden
+   */
+  HIDDEN = 'hidden',
```
Comment (dpkjnr) [inline modules/statics/src/base.ts:298]: Name is very ambiguous, when and where is it hidden?

**Review by dpkjnr — COMMENTED** [review]
You haven't added hidden coin feature to any coin, will it be done in upcoming PRs?

**Comment by ranger2255** [comment]
> You haven't added hidden coin feature to any coin, will it be done in upcoming PRs?

yes thats correct

File: modules/statics/src/base.ts (hunk 1 above)
Comment (ranger2255) [reply modules/statics/src/base.ts:298]: feature to be used from UI to hide a coin/token

Comment (margueriteblair) [reply modules/statics/src/base.ts:298]: I'd agree and say name should be more explicit, or at least in the comment you can add `This coin is hidden from the UI`

Comment (ranger2255) [reply modules/statics/src/base.ts:298]: As discussed with @dpkjnr, we ll go with the existing DEPRECATED coin feature to hide gteth from gas tank; 
steps:
1. create an alpha version with deprecated feature for gteth
2. use this version to fix tests in wp
3. attach evidences from (2) here and merge this PR
4. create a PR for wp with test fixes

**Pull request closed without merging**